pages/validator/external-chains/optimism.md

```diff
@@ -0,0 +1,440 @@
+```bash
+docker compose logs fault-detector -f
+```
+```bash
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386539264,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 2.639 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386554265,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 3.836 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386569266,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 3.624 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386584265,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 3.928 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386599264,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 2.452 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386614265,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 3.352 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386629264,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 2.516 ms\n","msg":"server log"}
+simple-optimism-node-fault-detector-1  | {"level":30,"time":1674386644264,"log":"172.22.0.7 - GET /metrics HTTP/1.1 200 - - 2.639 ms\n","msg":"server log"}
+```
+
+## Verify
+
+#### Method 1:
+```bash
+YOUR_IP=$(curl -4 ifconfig.co)
+curl -X POST $YOUR_IP:9991 -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' | jq
+
+```
+If you get something like this in response of the above rpc call, your node is setup correctly
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "0x3df827"
+}
+```
+You can use a [hex to number convertor](https://www.binaryhexconverter.com/hex-to-decimal-converter) to get the block height
+
+
+#### Method 2:
+You can also check your status by connecting to geth console
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+docker exec -it simple-optimism-node-l2geth-1 geth attach http://localhost:8545
+eth.blockNumber
+400040 # it will show you the latest block
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+docker exec -it simple-optimism-node-op-geth-1 geth attach http://localhost:8545
+eth.blockNumber
+4000400 # it will show you the latest block
+`}
+</CodeBlock>
+}
+]} />
+
+You can compare the block height on your node with explorer ([mainnet](https://optimistic.etherscan.io) or [testnet](https://goerli-optimism.etherscan.io)), use your RPC only when it has caught up with the latest block height
+
+## Endpoint
+
+```bash
+echo "${YOUR_IP}:9991"
+```
+
+### Configure vald
+
+In order for `vald` to connect to your Optimism node, your `rpc_addr` should be exposed in
+vald's `config.toml`
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="yaml">
+{`
+[[axelar_bridge_evm]]
+name = "optimism"
+l1_chain_name = "Ethereum"
```

Comment:
`l1_chain_name` is not needed for optimism. Validators just need to ensure that their L1 node settings for optimism node itself point to their ethereum rpc node.

Reply:
ahh sure, I just copied the Arbritrum settings - thinking it would be similar.

Reply:
yeah, bedrock upgrade simplified finality checks, so we don't need it anymore